draft-ietf-mls-architecture.md

```diff
@@ -561,7 +559,8 @@ weakening the PCS guarantees for attachments.
 
 In general we do not consider Denial of Service (DoS) resistance to be the responsibility
 of the protocol. However, it should not be possible for anyone to perform a
-trivial DoS attack from which it is hard to recover.
+trivial DoS attack from which it is hard to recover. This excludes the DS, which
```

Comment:
"This excludes the DS" -> "This excludes the DS as an attacker"

Reply:
You can revert this change. "Anyone" is already contains the DS.

Reply:
Oh sorry @beurdouche, I meant that the DS is able to DoS, as opposed to anyone else who shouldn't be able to. Will update to @raphaelrobert's suggestion.